crates/wash-cli/src/app/mod.rs

```diff
@@ -217,6 +217,16 @@ pub async fn handle_command(
 
     Ok(command_output?)
 }
+/// Validate a Wadm manifest file
+async fn validate_handler(cmd: ValidateCommand) -> Result<CommandOutput> {
```

Comment:
SUPER NIT: we use `handle_<something>` for most of our functions, would you mind a rename here?